--- open-vmdk-875f4162c91c7fc7bc450dccaf7b896a927fa42b/vmdk/diskinfo.h.orig	2021-11-04 07:17:03.000000000 +0100
+++ open-vmdk-875f4162c91c7fc7bc450dccaf7b896a927fa42b/vmdk/diskinfo.h	2022-01-05 20:54:21.194950226 +0100
@@ -35,7 +35,7 @@ struct DiskInfo {
 	const DiskInfoVMT *vmt;
 };
 
-char *toolsVersion; /* toolsVersion in metadata */
+extern char *toolsVersion; /* toolsVersion in metadata */
 
 DiskInfo *Flat_Open(const char *fileName);
 DiskInfo *Flat_Create(const char *fileName, off_t capacity);
--- open-vmdk-875f4162c91c7fc7bc450dccaf7b896a927fa42b/vmdk/mkdisk.c.orig	2021-11-04 07:17:03.000000000 +0100
+++ open-vmdk-875f4162c91c7fc7bc450dccaf7b896a927fa42b/vmdk/mkdisk.c	2022-01-05 20:55:53.239767639 +0100
@@ -24,6 +24,8 @@
 #include <string.h>
 #include <getopt.h>
 
+char *toolsVersion; /* toolsVersion in metadata */
+
 static int
 copyData(DiskInfo *dst,
          off_t dstOffset,
